packages/itwinui-css/src/surface/classes.scss

```diff
@@ -5,3 +5,20 @@
 .iui-surface {
   @include iui-surface-component;
 }
+
+.iui-surface-header {
+  @include iui-surface-header-footer(header);
```

Comment:
Might be better to create intermediate mixins so the logic stays out of `classes`.

like this:
```scss
@mixin iui-surface-header {
  @include iui-surface-header-footer(header);
}
```
```scss
@mixin iui-surface-footer {
  @include iui-surface-header-footer(footer);
  @include iui-surface-footer-modifiers;
}
```

Reply:
I've done this.